packages/react-server/src/ReactFizzServer.js

```diff
@@ -305,7 +306,17 @@ export function createRequest(
     onShellError: onShellError === undefined ? noop : onShellError,
     onFatalError: onFatalError === undefined ? noop : onFatalError,
   };
-  // This segment represents the root fallback.
+  const rootBoundaryID = getRootBoundaryID(responseState);
```

Comment:
`responseState` isn't really supposed to be where all configuration lives. It represents things that are stateful. We kind of abused it to also store cached things (converted to chunks).

In this case you don't really need to create a response state only to read back the value from it immediately. You can just pass in a SuspenseBoundaryID as the argument to `createRequest` here to initialize the root boundary.